librarymanagement/src/main/java/org/librarymanagement/exception/DuplicateFieldException.java

```diff
@@ -0,0 +1,15 @@
+package org.librarymanagement.exception;
+
+
+import lombok.Getter;
+
+import java.util.Map;
+
+@Getter
+public class DuplicateFieldException  extends RuntimeException{
```

Comment:
```suggestion
public class DuplicateFieldException extends RuntimeException{
```
đây gọi là skill 8 năm reviewer